utils/appliance/__init__.py

```diff
@@ -58,6 +58,31 @@ class ApplianceException(Exception):
     pass
 
 
+class ApplianceConsoleCli(object):
+
+    def __init__(self, appliance):
+        self.appliance = appliance
+
+    def run(self, ap_cli_command):
+        return self.appliance.ssh_client.run_command(
+            "appliance_console_cli {}".format(ap_cli_command))
+
+    def set_hostname(self, hostname):
+        self.run("-H {}".format(hostname))
+
+    def configure_appliance_internal_fetch_key(self, region, dbhostname,
+            username, password, dbname, fetch_key, sshlogin, sshpass):
+        self.run("-r {} -i -h {} -U {} -p {} -d {} -v -K {} -s {} -a {}".format(
+            region, dbhostname, username, password, dbname, fetch_key, sshlogin, sshpass))
```

Comment:
Looking at this, we might like to use `utils.quote.quote` to ensure the parameters get passed correctly to the shell. I was thinking about something like this:

```python
from utils.quote import quote as shell_quote
def run(self, command, *args, **kwargs):
    args = [shell_quote(str(arg)) for arg in args]
    kwargs = {k: shell_quote(str(v)) for k, v in kwargs.items()}
    formatted_command = command.format(*args, **kwargs)
    return self.appliance.ssh_client.run_command("appliance_console_cli {}".format(formatted_command))
```

Then the usage would be similar to logger:
```python
self.run('-r {} -i -h {} -U {} -p {} -d {} -v -K {} -s {} -a {}', region, dbhostname, username, password, dbname, fetch_key, sshlogin, sshpass)
```

@psav @RonnyPfannschmidt ?